packages/ovm-toolchain/test/test-truffle/erc20.spec.js

```diff
@@ -1,7 +1,7 @@
-const EIP20Abstraction = artifacts.require('EIP20');
+const EIP20Abstraction = artifacts.require('ERC20');
```

Comment:
Thank you for finally making this change 😂

Reply:
Lol I had to revert back because the `ERC20` contract didn't emit events. I'm gonna fix that in a sec though.